tests/Adapter/Phpunit/Process/PhpunitExecutableFinderTest.php

```diff
@@ -20,4 +20,9 @@ public function testFinderCanLocatePhpunitExecutable()
         $result = $finder->find();
         $this->assertRegExp('%phpunit.*(\\.bat|\\.phar)?$%i', $result);
     }
+
+    public function testFinderCanLocateACustomPhpunitNotInVendorOrPath()
+    {
+        $this->markTestIncomplete('Design test to inject custom config for this.');
```

Comment:
Leaving this aside for the time being. There's no clean way to inject the config change needed at the moment.